receiver/postgresqlreceiver/client.go

```diff
@@ -458,9 +458,9 @@ func (c *postgreSQLClient) getReplicationStats(ctx context.Context) ([]replicati
 	query := `SELECT
 	client_addr,
 	coalesce(pg_wal_lsn_diff(pg_current_wal_lsn(), replay_lsn), -1) AS replication_bytes_pending,
-	coalesce(write_lag, -1),
-	coalesce(flush_lag, -1),
-	coalesce(replay_lag, -1)
+	extract('epoch' from coalesce(write_lag, '-1 seconds')),
+	extract('epoch' from coalesce(flush_lag, '-1 seconds')),
+	extract('epoch' from coalesce(replay_lag, '-1 seconds'))
```

Comment:
@dehaansa, do you mind double checking that `epoch` can be quoted here? I've found many examples without the quotes, none with - just want to be sure.

Reply:
I tested the query as written against a non-replication postgres DB and it ran successfully. I also tested using `extract('epoch'...` in a little test table I made with an interval that had data, and it reported what I expected it to.